Resources/Prototypes/_NF/Entities/Structures/Doors/Airlocks/shuttle.yml

```diff
@@ -0,0 +1,37 @@
+- type: entity
+  id: AirlockShuttleAssemblyNfsd
+  parent: AirlockShuttleAssembly
+  name: external airlock assembly
+  suffix: Docking
+  description: An incomplete structure necessary for connecting two space craft together.
+  components:
+  - type: Transform
+    anchored: true
+    noRot: false
+  - type: Rotatable
+  - type: Sprite
+    sprite: Structures/Doors/Airlocks/Glass/shuttle.rsi
+    state: closed
```

Comment:
Most of this could be removed because of your parenting - might want to refer to your green sprite though.